.changelog/4707.internal.md

```diff
@@ -0,0 +1,4 @@
+ci: Build OpenSSL statically for Oasis Core releases
+
+This makes sure that oasis-core-loader is more portable between
```

Comment:
This seems questionable as hell, though OpenSSL is moderately better than it used to be.  Though still not something I would personally staticly link ever, because getting updates from the OS vendor is a good idea.

Reply:
This is specifically for the oasis-core-runtime-loader (the SGXS loader) so I don't see any problems here. And it does make it easier to use the binary.

Reply:
I guess, I'm not thrilled by this though.